packages/cw3/src/query.rs

```diff
@@ -31,20 +31,20 @@ pub enum Cw3QueryMsg {
     /// Query the vote made by the given voter on `proposal_id`. This should
     /// return an error if there is no such proposal. It will return a None value
     /// if the proposal exists but the voter did not vote. Returns VoteResponse
-    Vote { proposal_id: u64, voter: HumanAddr },
+    Vote { proposal_id: u64, voter: Addr },
```

Comment:
All `*Msg` types should use `String` (this is unverified input from users)
All state should use `Addr`.

By doing so, the compiler should enforce we validate input before saving it